posixlib/src/main/scala/scala/scalanative/posix/netinet/in.scala

```diff
@@ -17,17 +17,20 @@ object in {
   type sockaddr_in = CStruct4[socket.sa_family_t, // sin_family
                               in_port_t,        // sin_port
                               in_addr,          // sin_addr
-                              CArray[Byte, _8]] // sin_zero, Posix allowed
+                              CArray[Byte, _8] // sin_zero, Posix allowed
+  ]
```

Comment:
Can we try to revert that, it looks a bit weird I think. I think in Scala.js ending square bracket sticks to the right side of the last type.

Reply:
These will all be indented left in a more Scala 3 style so it won't look like it is hanging out there.